modelzoo/WnD/TensorFlow2/README.md

```diff
@@ -0,0 +1,71 @@
+# Intel Optimized Wide and Deep
+
+## pre-work
+sync submodule code
+```
+git submodule update --init --recursive
+```
+
+apply patch
+```
+cd modelzoo/WnD/TensorFlow2
+bash patch_wnd.patch
+```
+
+Source repo: https://github.com/NVIDIA/DeepLearningExamples/tree/master/TensorFlow/Recommendation/WideAndDeep
+
+## Model
+
+Google's [Wide & Deep Learning for Recommender Systems](https://arxiv.org/abs/1606.07792)
+
+## Environment setup
+
```

Comment:
what is this guide for? bare-metal environment? 
Are you expecting the user to install those packages all by themselves or should use our docker file? 
Please specifically called this out - we are providing docker files, but here you are providing a bare-metal installation?  
It doesn't match the overall installation guide.